openpgp/keys.go

```diff
@@ -442,6 +452,13 @@ EachPacket:
 			current.Name = pkt.Id
 			current.UserId = pkt
 		case *packet.Signature:
+			if pkt.SigType == packet.SigTypeKeyRevocation {
+				// These revocations won't revoke UIDs (see
+				// SigTypeIdentityRevocation). Handle these first,
+				// because key might have revocation coming from
+				// another key (designated revoke).
+				revocations = append(revocations, pkt)
```

Comment:
That's not my question.  My question is that now these packets are going to hit line 551 in the new code, whereas previous it did not (since it was `if .. else` gated and handled by a different condition).  So all of these packets will be in the `current.Signatures` array as a result of line 551.  Does that matter?